runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1609,14 +1609,14 @@ TR_JITServerRelocationRuntime::allocateSpaceInDataCache(uintptr_t metaDataSize,
    }
 
 uint8_t *
-TR_JITServerRelocationRuntime::copyDataToCodeCache(const void *startAddress, size_t totalSize, TR_J9VMBase *fe)
+TR_JITServerRelocationRuntime::copyDataToCodeCache(const void *startAddress, size_t totalSize, TR_J9VMBase *fe, TR::Compilation *comp)
```

Comment:
It's not clear why the `TR::Compilation` object is being passed in this case rather than the `TR::CodeCacheKind` as was done everywhere else.

If the compilation object really is what's needed, I notice that code in this method is able to get to that object via `compInfoPT->getCompilation()`.  Could the extra argument be dropped entirely in that case?